app/code/community/Bolt/Boltpay/Model/Order.php

```diff
@@ -329,7 +279,69 @@ public function createOrder($reference, $sessionQuoteId = null, $isPreAuthCreati
         return $order;
     }
 
+    /**
+     * @param $immutableQuote
+     * @param $transaction
+     * @param $isPreAuthCreation
+     * @return bool
+     * @throws Exception
+     */
+    protected function applyShipmentToQuoteFromTransaction($immutableQuote, $transaction, $isPreAuthCreation){
+        $packagesToShip = @$transaction->order->cart->shipments;
+
+        if (!$packagesToShip) {
+            return false;
+        }
```

Comment:
@johnmccombs1 
As I see it, this is the relevant change, error suppression and refactoring to an early return.  Is this correct?

Reply:
@mccottry Yes. That's right